package.json

```diff
@@ -43,38 +43,36 @@
     "qs": "^6.6.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.1.2",
-    "@babel/plugin-proposal-class-properties": "^7.1.0",
-    "@babel/preset-env": "^7.1.0",
+    "@babel/core": "^7.2.2",
+    "@babel/plugin-proposal-class-properties": "^7.3.0",
+    "@babel/preset-env": "^7.3.1",
     "@babel/preset-typescript": "^7.1.0",
-    "@types/jest": "^23.3.9",
-    "@types/lodash": "^4.14.117",
+    "@types/jest": "^23.3.13",
+    "@types/lodash": "^4.14.120",
     "@types/qs": "^6.5.1",
-    "axios-mock-adapter": "^1.15.0",
+    "axios-mock-adapter": "^1.16.0",
     "babel-core": "7.0.0-bridge.0",
     "babel-jest": "^23.6.0",
-    "commitizen": "^3.0.4",
+    "commitizen": "^3.0.5",
     "cz-conventional-changelog": "^2.1.0",
-    "dotenv": "^6.1.0",
-    "husky": "^1.1.3",
+    "dotenv": "^6.2.0",
+    "husky": "^1.3.1",
     "jest": "^23.6.0",
     "jest-cli": "^23.6.0",
-    "lint-staged": "^8.0.4",
-    "prettier": "^1.14.3",
-    "rollup": "^0.66.6",
+    "lint-staged": "^8.1.0",
+    "prettier": "^1.16.1",
+    "rollup": "^1.1.2",
+    "rollup-plugin-copy-glob": "^0.3.0",
     "rollup-plugin-includepaths": "^0.2.3",
     "rollup-plugin-json": "^3.1.0",
-    "rollup-plugin-node-resolve": "^3.4.0",
-    "rollup-plugin-typescript2": "^0.17.2",
-    "rollup-plugin-url": "^2.0.1",
-    "ts-jest": "^23.10.4",
-    "ts-node": "^7.0.1",
+    "rollup-plugin-node-resolve": "^4.0.0",
+    "rollup-plugin-typescript2": "^0.19.2",
+    "ts-jest": "^23.10.5",
+    "ts-node": "^8.0.1",
     "tslib": "^1.9.3",
-    "tslint": "^5.11.0",
+    "tslint": "^5.12.1",
```

Comment:
Can we remove `tslint` and start using the [TypeScript ESLint plugin](https://github.com/typescript-eslint/typescript-eslint)?

https://eslint.org/blog/2019/01/future-typescript-eslint

Reply:
Just came across the exact same article. It looks like the better option indeed! Let's migrate to ESLint then.

Reply:
Turned out to be a great idea! I have updated the [eslint repo](https://github.com/mollie/eslint-config-mollie).